aduro/__init__.py

```diff
@@ -0,0 +1,26 @@
+"""
+Aduro Stove on Seluxit Echidna proxy that communicates via Wappsto API
+"""
+from .version import __version__
+
+
+class Aduro:
+    """Aduro Stove proxi client"""
```

Comment:
```suggestion
    """Aduro Stove proxi client."""
```

Reply:
Doesn't the linting object to this?

Reply:
@theovald strangely not 😢 ... need better linting 🤷‍♀️